_data/perf.toml

```
@@ -0,0 +1,28 @@
+# Example of a homepage performance card.
+# Both `title` and `description` can contain markdown for styles, but not links.
+# [[dashboards]]
+# title = "GETs/SETs on `9.0.0`"
+# description = "Performance dashboard of GETs and SETs on the Valkey 9.0.0 release. Updated on Oct 21, 2025."
+# link = "https://perf-dashboard.valkey.io/public-dashboards/3e45bf8ded3043edaa941331cd1a94e2"
+
+# Order in this file indicates rendering order.
+
+[[dashboards]]
+title = "Valkey Performance Dashboards"
+description = "Valkey Performance Dashboards provide a consolidated view of throughput and memory efficiency trends across versions, helping teams validate improvements and identify regressions."
+link = "/performance/"
+
+# Performance dashboard sections for /performance page
+# Order in this file indicates rendering order.
+
+[[sections]]
+title = "Throughput Across Versions"
+iframe_url = "https://df0m5orgq2d38.cloudfront.net/public-dashboards/e6e8ec1f88954ddd8a67df1ab4966eb2"
+description = "This dashboard visualizes throughput trends across Valkey versions. It helps compare key releases side by side, highlight performance gains from new features. Use it when evaluating whether a new version meets your performance SLOs before rollout."
+methodology = "These metrics are generated using the valkey-perf-benchmark tool on a c8g.metal.2xl instance, running a matrix of configurations that vary pipelining (1, 10), I/O threading (1, 9), and data sizes 512 bytes. To further stabilize results, we apply IRQ tuning by pinning network interrupts away from CPUs dedicated to the server process, and we isolate the server and benchmark client on separate NUMA nodes to remove L3 cache contention. This controlled methodology ensures consistent, repeatable comparisons across versions and makes performance gains or regressions easier to attribute to real code changes rather than environmental variability."
+
+[[sections]]
+title = "Memory Overhead Across Versions"
+iframe_url = "https://df0m5orgq2d38.cloudfront.net/public-dashboards/1e3d87a62b794ce9aa81281d8399e4bf"
+description = "This dashboard visualizes Memory Efficiency trends across Valkey versions. It helps compare releases side by side, highlight Memory Efficiency gains from new features. Use it when evaluating whether a new version meets your performance SLOs before rollout."
```

```suggestion
description = "This dashboard visualizes Memory Efficiency trends across Valkey versions. It helps compare releases side by side, highlight Memory Efficiency gains from new features."
```

What Performance SLO?